.gitignore

```diff
@@ -1,3 +1,4 @@
 /bin/
 /vendor/
-composer.lock
\ No newline at end of file
+composer.lock
+/nbproject/private/
```

Comment:
Please use a global gitignore file for such settings. Please revert to the old contents.